test/features/environment.py

```diff
@@ -17,7 +17,7 @@
     'Admin': dict(
         name=u'admin',
         email=u'admin@localhost',
-        password=u'password'
+        password=u'Password123!'
```

Comment:
pull in from environment, don't make it the same all the time just in case people find the box that was stood up and 'do stuff'